From 0b926132265100d50cfb3f8a00d37afd76e243e7 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Mon, 31 Oct 2016 23:22:11 -0700
Subject: [PATCH 8/9] libunwind: Silence -Wundef warning in libunwind.h

libunwind/include/libunwind.h:31:7: warning: '__arm__' is not defined, evaluates to 0 [-Wundef]
  #if __arm__
      ^
1 warning generated.

https://llvm.org/bugs/show_bug.cgi?id=30861

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
(cherry picked from commit 58844e470ee2fb46b0bb4bb6968742f90e8c6486)
---
 include/libunwind.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/libunwind.h b/include/libunwind.h
index 34c40de..90c5595 100644
--- a/include/libunwind.h
+++ b/include/libunwind.h
@@ -28,7 +28,7 @@
     #include <Availability.h>
   #endif
 
-  #if __arm__
+  #ifdef __arm__
      #define LIBUNWIND_AVAIL __attribute__((unavailable))
   #elif defined(__OSX_AVAILABLE_STARTING)
     #define LIBUNWIND_AVAIL __OSX_AVAILABLE_STARTING(__MAC_10_6, __IPHONE_5_0)
-- 
2.10.1

